### ISSUE TITLE
Meta Description for Login page

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-cms#86]
_From @adamjohnson on January 7, 2015 16:18_

Here's the login page if you search for CleanSlate on Google:

![https://s3.amazonaws.com/f.cl.ly/items/1l2E383n1d1p3W131w0K/meta-description.png](https://s3.amazonaws.com/f.cl.ly/items/1l2E383n1d1p3W131w0K/meta-description.png)

I'd like to add the following meta description to the Login page:

```html
<meta name="description" content="Log in to CleanSlate, West Virginia University’s web content management system.">
```

This should be pretty simple to do with layouts (FTW!).

Hat tip to Sarah Gould for helping with the copy.

_Copied from original issue: wvuweb/cleanslate#85_

**Comment by adamjohnson** [comment]
_From @zeroedin on January 7, 2015 16:27_

https://github.com/mcasimir/meta-tags-helpers

Issue with it is codebase hasn't been worked on in a year.

**Comment by adamjohnson** [comment]
_From @nreckart on January 8, 2015 15:45_

Implemented in c7cf159492d4ed.
